src/components/vouchers/VoucherRedeem.tsx

```diff
@@ -67,15 +81,15 @@ export const VoucherRedeem = ({ quota, voucherId }: { quota: TQuota; voucherId:
   const seconds = remainingTime % 60;
 
   const redeemVoucher = async () => {
-    try {
-      let deviceToken = await readFromStore(VOUCHER_DEVICE_TOKEN);
+    let deviceToken = await readFromStore(VOUCHER_DEVICE_TOKEN);
 
-      if (!deviceToken) {
-        deviceToken = uuid();
-        addToStore(VOUCHER_DEVICE_TOKEN, deviceToken);
-      }
+    if (!deviceToken) {
+      deviceToken = uuid();
+      addToStore(VOUCHER_DEVICE_TOKEN, deviceToken);
+    }
 
-      redeemQuotaOfVoucher({
+    try {
+      await redeemQuotaOfVoucher({
```

Comment:
why was the `await` added here?

Reply:
I think it is an async function, so I thought it would make more sense to use it with `await`.

Reply:
but are you awaiting something in order to go on with the result? i don't think so. this can be called and done, i guess.

Reply:
so far it's worked without `await`. I'll delete it and we'll see what happens.

Reply:
sure, the method is still called ;)

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1104/commits/888657a5fc0230d15449fb430c961ae348791255